pkg/cmd/chat/reaction/reaction.go

```diff
@@ -0,0 +1,133 @@
+package reaction
+
+import (
+	stream_chat "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		getCmd(),
+		sendCmd(),
+		deleteCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-reactions [message-id]",
+		Short: "Get reactions for a message",
+		Example: heredoc.Doc(`
+			# Get reactions for a [08f64828-3bba-42bd-8430-c26a3634ee5c] message
+			$ stream-cli chat get-reactions 08f64828-3bba-42bd-8430-c26a3634ee5c --output-format json
+		`),
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			msgId := args[0]
+
+			resp, err := c.Channel("", "").GetReactions(cmd.Context(), msgId, nil)
+			if err != nil {
+				return err
+			}
+
+			return utils.PrintObject(cmd, resp.Reactions)
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringP("output-format", "o", "json", "[optional] Output format. Can be json or tree")
+
+	return cmd
+}
+
+func sendCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "send-reaction --message-id [message-id] --user-id [user-id] --reaction-type [reaction-type]",
+		Short: "Send a reaction to a message",
+		Long: heredoc.Doc(`
+			Stream Chat has built-in support for user Reactions. Common examples are
+			likes, comments, loves, etc. Reactions can be customized so that you
+			are able to use any type of reaction your application requires.
+		`),
+		Example: heredoc.Doc(`
+			# Send a reaction to a [08f64828-3bba-42bd-8430-c26a3634ee5c] message
+			$ stream-cli chat send-reaction --message-id 08f64828-3bba-42bd-8430-c26a3634ee5c --user-id 12345 --reaction-type like
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			reactionType, _ := cmd.Flags().GetString("reaction-type")
+			msgID, _ := cmd.Flags().GetString("message-id")
+			userID, _ := cmd.Flags().GetString("user-id")
+
+			r := &stream_chat.Reaction{Type: reactionType}
+
+			_, err = c.Channel("", "").SendReaction(cmd.Context(), r, msgID, userID)
+			if err != nil {
+				return err
+			}
+
+			cmd.Println("Successfully sent reaction")
+			return nil
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringP("reaction-type", "r", "", "[required] The reaction type to send")
```

Comment:
how about custom data and score ?

Reply:
I wouldn't expect people would go for it immediately, could be nicer since we support in messages. We could add it later too  if asked

Reply:
added score